transport/rif_comms/node.py

```diff
@@ -1,16 +1,20 @@
 from typing import Any
 
 from raiden.message_handler import MessageHandler
+from raiden.messages import Message
 from raiden.raiden_service import RaidenService
+from raiden.utils.runnable import Runnable
 from raiden.utils.typing import Address
-from transport.message import Message
 from transport.node import Node as TransportNode
 
 
-class RifCommsNode(TransportNode):
+class RifCommsNode(TransportNode, Runnable):
 
     def __init__(self, address: Address, config: dict):
         TransportNode.__init__(self, address)
+        Runnable.__init__(self)
+        self._config = config
+        print("RifCommsNode init on grpc endpoint: {}".format(self._config["grpc_endpoint"]))
```

Comment:
what about a log entry here instead of `print`?

Reply:
this must be removed, its just for dev test